.github/workflows/unit-test-object.yml

```diff
@@ -11,14 +11,34 @@ on:
 
 jobs:
   tests:
-    name: "Object Service ${{ matrix.object }} - WP: ${{ matrix.wp_version }} - PHP: ${{ matrix.php }}"
     strategy:
       fail-fast: false
       matrix:
-        php: ['8.3']
-        wp_version: ['trunk', 'latest', '6.1']
-        multisite: [true, false]
-        object: ["false", "memcached", "redis"]
+        include:
+          - php: '8.3'
+            wp_version: 'trunk'
+            object: 'redis'
+            multisite: true
+          - php: '8.3'
+            wp_version: 'trunk'
+            object: 'memcached'
+            multisite: false
+          - php: '8.2'
+            wp_version: '6.4'
+            object: ''
+            multisite: false
+          - php: '8.2'
+            wp_version: 'latest'
+            object: ''
+            multisite: false
+          - php: '8.1'
+            wp_version: 'latest'
+            object: ''
+            multisite: false
```

Comment:
PHP 8.1 and 8.2 are used the most by WordPress installations: https://wordpress.org/about/stats/